content/en/serverless/google_cloud_run/_index.md

```diff
@@ -2,201 +2,234 @@
-### Build your container
+## Getting started
+
+### Prerequisites
+
+Make sure you have a [Datadog API Key][10] and are using a programming language [supported by a Datadog tracing library][2].
+
+### Deploy a sample application in one click
+
+To deploy a sample application without the need to follow the rest of the guide, you can use on of [these examples][12]. The installation process will ask the required details such as the Datadog API Key. Note that these applications send data to the default website `datadoghq.com`.
 
-If you are using a Dockerfile to build your application, complete the following:
+### Configure your container with the Datadog Agent
 
-1. Instrument your application with a [supported Datadog tracing library][2]
+To build your container with Datadog instrumentation you can follow one of these two methods, depending if you are using a Dockerfile or buildpack.
 
-2. Use the `COPY` instruction to copy the [Datadog `serverless-init` binary][3] into your Docker image.
+#### Configure the Datadog Agent using Dockerfile
 
-3. Use the `ENTRYPOINT` instruction to run the `serverless-init` binary as your Docker container is initiated.
+Using a Dockerfile to build your container, the following steps are needed:
 
-4. Use the `CMD` instruction to run your existing application and other required commands as arguments.
+1. Use the `COPY` instruction to copy the [Datadog `serverless-init` binary][3] into your Docker image.
 
-The following are examples of how to complete these three steps. You may need to adjust these examples depending on your existing Dockerfile setup. 
+2. Use the `ENTRYPOINT` instruction to run the `serverless-init` binary as your Docker container is initiated.
 
+3. Use the `CMD` instruction to run your existing application and other required commands as arguments.
+
+You can accomplish those steps by adding the following lines to your Dockerfile. You may need to adjust these examples depending on your existing Dockerfile setup.
 
 {{< programming-lang-wrapper langs="go,python,nodejs,java,dotnet,ruby" >}}
 {{< programming-lang lang="go" >}}
+
 ```
-COPY --from=datadog/serverless-init:beta4 /datadog-init /app/datadog-init
+COPY --from=datadog/serverless-init /datadog-init /app/datadog-init
 ENTRYPOINT ["/app/datadog-init"]
 CMD ["/path/to/your-go-binary"] (adapt this line to your needs)
 ```
-See [Tracing Go Applications][1] for more details. [Sample code for a simple Go application][2].
 
+[Sample code for a simple Go application](https://github.com/DataDog/crpb/tree/main/go).
 
-[1]: /tracing/setup_overview/setup/go/?tabs=containers
-[2]: https://github.com/DataDog/crpb/tree/main/go
 {{< /programming-lang >}}
-
 {{< programming-lang lang="python" >}}
 
 ```
-COPY --from=datadog/serverless-init:beta4 /datadog-init /app/datadog-init
+COPY --from=datadog/serverless-init /datadog-init /app/datadog-init
 ENTRYPOINT ["/app/datadog-init"]
 CMD ["ddtrace-run", "python", "app.py"] (adapt this line to your needs)
```

Comment:
Actually I think we should pull the key changes, e.g., installing the tracer, to the sample dockerfile here. Otherwise user need to spend a good amount of time looking at the sample repo to know what are part of the sample app vs what are the datadog changes they need to add to their own app. Also here we already give instruction using "ddtrace-run", but there is no instruction about installing ddtrace, which is really strange and confusing.

Reply:
it's not for all languages, I added the installation when needed